Cargo.lock

```diff
@@ -395,7 +395,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "freetype"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
This update happened automatically, I believe it is due to https://github.com/servo/webrender/commit/d16294236e0849ba0337815944628df78ec4741f

Reply:
Doesn't webrender have a CI check for that ?

Reply:
I guess not? Though I don't know, I haven't contributed to webrender before. I've opened a separate PR to fix this: https://github.com/servo/webrender/pull/2599